src/docs/installing/distros.md

```diff
@@ -145,4 +153,4 @@ boot.extraModprobeConfig = ''
 boot.extraModprobeConfig = ''
   options snd-intel-dspcfg dsp_driver=4
 '';
-```
+```
```

Comment:
No newline?

Reply:
originally i planned on putting a tip down there, i moved it back to the top.